package.json

```diff
@@ -6,12 +6,13 @@
     "apps": "./script/app-list.sh",
     "analyze": "webpack-bundle-analyzer build/vagovprod/generated/stats.json",
     "build": "node --max-old-space-size=4096 script/build.js",
-    "build-webpack": "NODE_OPTIONS=--max-old-space-size=4096 webpack --config config/standalone-webpack.config.js",
     "build-analyze": "NODE_ENV=production npm run build -- --buildtype vagovprod --analyzer; npm run analyze",
+    "build-content-test": "node script/test-cms-export.js",
```

Comment:
Is this idea of this test to compare the final Tome data structure against the GraphQL equivalent?

Reply:
Also, if we're gonna find ourselves creating a series of `content` commands, I think we should start using colons a bit for a namespace kind of thing. `npm run build:content:test`

Reply:
For right now the command just logs to the console some stuff for debugging, but I could definitely see it turning into that in the near-future.

Good thinking -- I much prefer the use of colons in commands tbh 👍